tests/tests_default.yml

```diff
@@ -2,6 +2,38 @@
 ---
 - name: Ensure that the role runs with default parameters
   hosts: all
-  gather_facts: false  # test that role works in this case
+  become: true
```

Comment:
it's better to omit `become` from playbooks - the user can provide `become` if necessary - on the command line, using env. var., using ansible.cfg, in the inventory, etc.